Feature/admin/rectification details

maksymilian-majer opened the pull request:
Closes #272

fjcero commented:
I use `import toJson from 'enzyme-to-json'` to reduce the size and improve readability of snapshots. If you will that is a good approach when updating the missing snapshot :)

maksymilian-majer commented:
@fjcero I rewrote to use `enzyme-to-json` and I think in most cases it's probably better to use it as default serializer. I added an issue #283 to consider if that's a good idea – there are also non-component snapshots after all.
